packages/jitm/tests/php/test_Manager.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Automattic\Jetpack\JITM;
+
+use Automattic\Jetpack\JITM\Manager as Jetpack_JITM;
+use phpmock\functions\FunctionProvider;
+use phpmock\Mock;
+use phpmock\MockBuilder;
+use PHPUnit\Framework\TestCase;
+
+class Test_Jetpack_JITM extends TestCase {
+	public function setUp() {
+		$builder = new MockBuilder();
+		$builder->setNamespace( __NAMESPACE__ )
+			->setName( 'add_action' )
+			->setFunction( function() {} );
+		$builder->build()->enable();
+	}
+
+	public function tearDown() {
+		Mock::disableAll();
+	}
+
+	public function test_jitm_disabled_by_filter() {
+		$this->mock_filters( array(
+			array( 'jetpack_just_in_time_msgs', false, false ),
+		) );
+
+		$this->assertFalse( Jetpack_JITM::init() );
+
+		$this->clear_mock_filters();
+	}
+
+	public function test_jitm_enabled_by_default() {
+		$this->mock_filters( array(
+			array( 'jetpack_just_in_time_msgs', false, true ),
+		) );
+
+		$this->assertTrue( ! ! Jetpack_JITM::init() );
+
+		$this->clear_mock_filters();
+	}
+
+	protected function mock_filters( $filters ) {
```

Comment:
should we create a package with these at some point?

Reply:
Yeah, definitely. I suggest, let's start seeing some patterns and think about what would be most useful to have there. There's not much tests yet here, so it'll take us some time to get there 😉